addons/hr_timesheet/views/hr_timesheet_menus.xml

```diff
@@ -6,6 +6,11 @@
               groups="group_hr_timesheet_user"
               web_icon="hr_timesheet,static/description/icon_timesheet.png"
     >
+        <menuitem id="timesheet_menu_activity_user"
+                  name="My Timesheets"
+                  groups="group_hr_timesheet_user"
```

Comment:
```suggestion
                  groups="group_hr_timesheet_user,!group_hr_timesheet_approver"
```

I think we could just display `My Timesheets` menu when the user is `Timesheet User` and not `Timesheet Approver`. 🤔 

What do you think? 🤔

Reply:
Oh in fact, it is already the case. 🙈